bootstrapform_jinja/templatetags/bootstrap.py

```diff
@@ -1,110 +1,148 @@
-from django import forms
+from django.forms import CheckboxInput, CheckboxSelectMultiple, \
+                         RadioSelect, FileInput
+from django.forms.boundfield import BoundField
 from django.template.loader import get_template
-from django_jinja import library
 from django.utils.safestring import mark_safe
+from django.utils.six import string_types
```

Comment:
I have a policy that all code written or edited since 1st Jan 2017 is python 3.5+ only. Is that ok with you?